frontend/src/app.js

```diff
@@ -2,6 +2,7 @@ import { Fragment } from "react";
 import { BrowserRouter as Router, Switch, Route } from "react-router-dom";
 import "semantic-ui-css/semantic.min.css";
 // Pages Import
+import ScrollTop from "./components/scrolltop/scrolltop";
```

Comment:
check #311 again.

 folder (component) must be inside the utils folder.

Reply:
@mohit355  Done!